commit 7ad0ccc9e64b89e23c4d9a184e4694d859122a06 (refs/changes/03/2403/8)
Author: Brittany Wald <brittany.wald@joyent.com>
Date:   2017-10-05T20:46:07+00:00 (2 years ago)
    
    MANTA-3371 triggers cannot be installed with any vnodes in read-only mode

diff --git a/package.json b/package.json
index a82fad1..490ff4a 100644
--- a/package.json
+++ b/package.json
@@ -33,7 +33,7 @@
         "mahi": "2.0.1",
         "marlin": "git+https://github.com/joyent/manta-marlin.git#master",
         "mime": "1.2.11",
-        "moray": "3.1.1",
+        "moray": "3.3.0",
         "once": "1.3.0",
         "restify": "2.6.3",
         "vasync": "^1.5.0",
